shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -279,13 +283,13 @@ public void popRoute() {
         mFlutterNavigationChannel.invokeMethod("popRoute", null);
     }
 
-    private void setTextScaleFactor(float textScaleFactor) {
+    private void setUserSettings() {
       Map<String, Object> message = new HashMap<>();
-      message.put("type", "systemSettings");
-      message.put("textScaleFactor", textScaleFactor);
-      mFlutterSystemChannel.send(message);
+      message.put("textScaleFactor", getResources().getConfiguration().fontScale);
```

Comment:
This is missing a the iOS implementation which you can fill in by using the `NSDateFormatter` configured using the current locale `[NSLocale currentLocale]`.

Reply:
Done.